### ISSUE TITLE
the fps of emu

### ISSUE THREAD

**Issue by sherlockgjh** [Varvrar/ios_logger#4]
I change the imuFreq, but the fps of imu is not changed. I want to consult you about how to change the fps of imu.

**Comment by Varvrar** [comment]
> 
> 
> I change the imuFreq, but the fps of imu is not changed. I want to consult you about how to change the fps of imu.

Yes, you changed right variable. imuFreq changes gyroUpdateInterval, accelerometerUpdateInterval and deviceMotionUpdateInterval. According Apple's documentation it should change fps of imu.
But "The value of this property is capped to minimum and maximum values; the maximum value is determined by the maximum frequency supported by the hardware". 
I dont know min value, but max = 100 for all devices as far as I know.
